src/docs/ganache/reference/cli-options.md

```diff
@@ -314,9 +314,7 @@ The alias `--gasPrice` has been deprecated, use `-g` instead.
 **`--miner.blockGasLimit`**, **`l`**
 
 === "Syntax"
-    ```bash
-    --miner.blockGasLimit=<STRING>
-    ```
+`bash --miner.blockGasLimit=<STRING> `
```

Comment:
This syntax doesn't work

Reply:
I realised earlier that I pushed some changes made by my code formatter :(

I've reverted those now.